test/test_specs.py

```diff
@@ -2648,6 +3095,22 @@ def test_composite_contains():
     assert ("a", ("b", ("c",))) in spec.keys(True, True)
 
 
+def get_all_keys(spec: TensorSpec, include_exclusive: bool):
+    """Given a TensorSpec, returns all exclusive and non-exclusive keys as a set tuples."""
```

Comment:
Let's add args and kwargs

Reply:
What do you mean precisely? do you want me to add the two *args and **kwargs arguments in the function signature? and what would they do?